api/docs/release.dox

```diff
@@ -149,6 +149,10 @@ Further non-compatibility-affecting changes include:
  - Added type_is_read() API that returns true if a trace type reads from memory.
  - Added instr_num_memory_read_access() and instr_num_memory_write_access() that return
    the number of memory read and write accesses of an instruction respectively.
+ - Deprecated #dr_fp_type_t for Floating-Point operation types amd migrate to
+   #dr_instr_category_t.
+   Deprecated instr_is_floating_ex(), replacing them with instr_is_floating_exten().
```

Comment:
Stale name instr_is_floating_exten() -- surprised Doxygen doesn't fail the build (parens trigger a link -- maybe not as mandatory as a # link so it silently skips?).